src/pybind/mgr/feedback/service.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+import json
+import requests
+from requests.exceptions import RequestException
+
+from .model import Feedback
+
+class config:
+    url = 'tracker.ceph.com'
+    port = 443
+
+class CephTrackerClient():
+
+    def list_issues(self):
+        '''
+        Fetch an issue from the Ceph Issue tracker
+        '''
+        headers = {
+            'Content-Type': 'application/json',
+        }
+        response = requests.get(
+            f'https://{config.url}/issues.json', headers=headers)
+        if not response.ok:
+            if response.status_code == 404:
+                raise FileNotFoundError
```

Comment:
`FileNotFoundError` can be misleading, IMHO `RequestException` is fine for a 404 error.
```suggestion
```

Reply:
FileNotFound it's the common error i returned for both cli and controller to cover 404 as both uses create_service from /feedback/service.py so not sure how i can separately deal with both cases.

Reply:
I don't see any reference to this exception in the controller. Can you give more details?

Reply:
Yes sorry this is just referenced in module.